src/modules/timeAgo/index.ts

```diff
@@ -1,6 +1,102 @@
-import TimeAgo from "./timeAgo";
+import { digitsFaToEn } from "../digits";
 
-export default (datetime: string): string => {
-	const timeago = new TimeAgo(datetime);
-	return timeago.getTimeAgo();
-};
+export function convertToTimeStamp(datetime: string): number {
```

Comment:
All your functions and critical code lines should have documentation by following the JSDoc rules.

Reply:
JSDoc rules added for all functions.